includes/polling/os/teleste.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$version = snmp_get($device, "1.3.6.1.2.1.47.1.1.1.1.10.10", "-OQv");
```

Comment:
Do you have a mib for these OIDs?

You will also need to convert them to snmp multi gets (see other files as examples).